stable/coscale/README.md

```diff
@@ -0,0 +1,60 @@
+# CoScale
+
+[CoScale](https://www.coscale.com/) offers full-stack monitoring for containers and microservices.
+
+## Introduction
+
+This chart adds the CoScale Agent to all nodes in your cluster via a DaemonSet.
+
+## Prerequisites
+
+- Kubernetes 1.2+ with Beta APIs enabled
```

Comment:
The installation requires daemonsets, which were introduced as a beta feature in Kubernetes 1.2: http://blog.kubernetes.io/2016/03/Kubernetes-1.2-even-more-performance-upgrades-plus-easier-application-deployment-and-management-.html